NetCoreDemos/Views/Common/DialogsAndNotificationsOverview.cshtml

```diff
@@ -9,14 +9,14 @@
     @foreach(var property in Model) {
         <div onclick="showPopup(@property.ID, '@property.Address')">
             <div class="item-content">
-                <img src="@property.Image" />
+                <img alt="@property.Address" src="@property.Image" />
                 <div class="item-options">
                     <div>
                         <div class="address">@property.Address</div>
                         <div class="price large-text">$@string.Format("{0:n0}", property.Price)</div>
                         <div class="agent">
                             <div id="@("house" + property.ID)">
-                                <img src="~/images/icon-agent.svg">
+                                <img alt="Listing agent" src="~/Content/Images/icon-agent.svg">
```

Comment:
Do we really need to change the path to svg here?

Reply:
Fixed: [e74e9aa0894fdc88f313789f8799b626ce473904].